index.html

```diff
@@ -2,6 +2,8 @@
 <html>
   <head>
     <link rel="stylesheet" href="/dist/tutor.css" />
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,300,300italic,400italic,900,700' rel='stylesheet' type='text/css'>
+    <link href='http://fonts.googleapis.com/css?family=Architects+Daughter' rel='stylesheet' type='text/css'>
```

Comment:
Should I combine with H1, h2, etc styles in a file called typography?
Or should typographic styles for things be rolled up into a whatever view they are included in (i.e. dashboard, ireading, etc.)?